### DISCUSSION TITLE
PnP Modern Search as an Isolated Web Part

### DISCUSSION THREAD

**Discussion by MrWorkingCat** [microsoft-search/pnp-modern-search#1912] [General]
I had a look at the AppManifest.xml in version 4.5.4.0 and saw IsDomainIsolated="false"
Is it possible (and is it in the plans of developers) to configure the PnP Modern Search v4 as an isolated web part?
https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/isolated-web-parts

**Comment by wobba** [comment]
No plans to focus on running them in an iframe, and from a security point of view there should not be a need for this as long as you don't run custom web components with code think can be malicious. We sanitize both the html/script and css to ensure a template cannot run random code or influence CSS outside of the web part it self.

* javascript is removed
* click handlers are removed
* css is prefixed to work inside the web part container
